src/storage/src/hummock/event_handler/uploader/mod.rs

```diff
@@ -965,7 +965,12 @@ impl UploaderData {
             .map(|task_id| {
                 let (sst, spill_table_ids) =
                     self.spilled_data.remove(task_id).expect("should exist");
-                assert_eq!(spill_table_ids, table_ids);
```

Comment:
What are `spill_table_ids` and `table_ids` referring to in this context?

Reply:
`spill_table_ids` is the involved table ids in a spill task. `table_ids` is the table ids to be synced.